UPGRADE.md

```diff
@@ -15,7 +15,7 @@ $e = new Error(
     'msg',
     null,
     null,
-    null,
+    [],
```

Comment:
This shouldn't be changed as it relates to migration to version `0.13.x`. Instead, we should have a new entry in Upgrade.md for this specific change.